package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "augur-core",
-  "verson": "0.1.0",
+  "version": "0.0.1",
```

Comment:
why the downgrade?

Reply:
Hm.  IMO we probably should only _bump_ the version, not downgrade it.

Reply:
I didn't downgrade it, the old fake version was spelled wrong ;-)

So I am just starting at 0.0.1 -- and then once we get things rolling we can pull up a real Version.